app/views/users/index.html.erb

```diff
@@ -28,6 +29,18 @@
                       <%= user.admin? ? 'Admin' : 'User' %>
                     <% end %>
                   </td>
+                  <td>
+                    <% if user.admin? %>
+                      Admins can only be deactivate by themselves
```

Comment:
a saját nevem mellett is ezt az üzenetet látom adminként. jó lenne valahogy a policy-ket újrahasználni, a megfelelő akció / üzenet kiírásához